test.py

```
@@ -9094,3 +9094,1163 @@ async def test_cli_config_validate_fails_when_unconfigured(db_pool):
+            """,
+            episode_id,
+        )
+        
+        assert summary is not None, "Episode summary not found"
+        assert summary["memory_count"] == 3, f"Expected 3 memories, got {summary['memory_count']}"
+        assert summary["first_memory_at"] is not None
+        assert summary["last_memory_at"] is not None
+        assert summary["first_memory_at"] <= summary["last_memory_at"]
+
+
+async def test_episode_time_range_gist_index(db_pool):
```

Ruff (0.14.10)

Redefinition of unused `test_episode_time_range_gist_index` from line 4635

(F811)